AsyncDisplayKit/ASDisplayNode+Beta.h

```diff
@@ -18,6 +18,23 @@ void ASPerformBlockOnMainThread(void (^block)());
 void ASPerformBlockOnBackgroundThread(void (^block)()); // DISPATCH_QUEUE_PRIORITY_DEFAULT
 ASDISPLAYNODE_EXTERN_C_END
 
+/**
+ * Bitmask to indicate what performance measurements the cell should record.
+ */
+typedef NS_OPTIONS(NSUInteger, ASDisplayNodePerformanceMeasurementOptions) {
+  ASDisplayNodePerformanceMeasurementOptionsNone = 0,
+  ASDisplayNodePerformanceMeasurementOptionsLayoutSpec = 1 << 0,
+  ASDisplayNodePerformanceMeasurementOptionsLayoutGeneration = 1 << 1
```

Comment:
Remove `ASDisplayNodePerformanceMeasurementOptionsNone` entry. `kNilOptions` (objc) or `[]` (Swift) is the standard for no options.